pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,33 +1,50 @@
 <section class="my-4">
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .Transaction.Txid }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Pending.EnvelopeId }}</span>
     </div>
   </section>
-  <form id="transaction-accept" hx-post="/v1/transactions/{{ .Transaction.Txid }}/send" hx-swap="none"
+  <form id="transaction-accept" hx-post="/v1/transactions/{{ .Pending.EnvelopeId }}/send" hx-swap="none"
```

Comment:
From what I can see, the `envelope ID` is in the pending object and that is why this change has been made.

Reply:
Oh I think I see the issue - so either a Transaction or a Pending object will be returned (one of, never both, never none). So possibly what we should do is have the backend create a top level TransactionID field (synonymous with EnvelopeID) and extract it from Transaction.EnvelopeID or Pending.EnvelopeID - which ever is available.

Note that `Transaction.Txid` refers to the ID of the transaction on the block chain and has nothing to do with the travel rule information exchange. (e.g. this is something that is simply part of the payload).